src/tests/build.proj

```diff
@@ -474,6 +474,7 @@
       <GroupBuildCmd>$(GroupBuildCmd) "/p:CLRTestBuildAllTargets=$(CLRTestBuildAllTargets)"</GroupBuildCmd>
       <GroupBuildCmd>$(GroupBuildCmd) "/p:__TestGroupToBuild=$(__TestGroupToBuild)"</GroupBuildCmd>
       <GroupBuildCmd>$(GroupBuildCmd) "/p:__SkipRestorePackages=1"</GroupBuildCmd>
+      <GroupBuildCmd>$(GroupBuildCmd) "/p:IsRuntimeBuild=True"</GroupBuildCmd>
```

Comment:
Did you check to see if there was an equivalent property defined elsewhere? It's nit-picky for sure :-)

Reply:
I wasn't able to determine exactly what property is designated as the "this is a build of dotnet/runtime" that would not be used in `Dotnet/Diagnostics` when building the `Microsoft.Diagnostics.NETCore.Client` project.
These were the available properties from the chain (`msbuild bigevent.csproj -> ResolveProjectReferences ->  msbuild common.csproj -> ResolveProjectReferences -> msbuild Microsoft.Diagnostics.NETCore.Client.csproj`) to build `Microsoft.Diagnostics.NETCore.Client` in my local dotnet/runtime https://gist.github.com/mdh1418/8029afc7706ff70a3f0cf916519799c7

Reply:
What if we added something like <Import Project="..\NetCoreClient.Customizations.props"/> to the Microsoft.Diagnostics.NetCore.Client.csproj file and then created this file in the runtime repo but left it absent in the diagnostics repo? I'm not at all an msbuild expert but I think that would work to let us make some repo-specific changes?

I'm hoping we can avoid messing with the repo-wide project files if possible.

Reply:
@hoyosjs - not sure if my suggestion is kosher or if you aware of better techniques?

Reply:
The root props always defines `GitHubRepositoryName` and `RepositoryUrl`. Would that be enough to identify this here?

Reply:
Thanks, I'll use the `GitHubRepositoryName` to condition these property and item groups. 
`'$(GitHubRepositoryName)' == 'runtime'` instead of `'$(IsRuntimeBuild)' == 'true'`